cypress/support/commands.js

```diff
@@ -23,3 +23,21 @@
 //
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+
+Cypress.Commands.add('findByPlaceholder', (placeholder) => {
+  cy.get(`[placeholder="${placeholder}"]`);
+});
+
+Cypress.Commands.add('login', (username = 'Dmytro', password = 'WordP@ss') => {
+  cy.request({
```

Comment:
In the 'login' command, you are not sending any data in the request body. Therefore, the server cannot authenticate the user. You should send the 'username' and 'password' in the request body.

Reply:
Fixed

Reply:
Thank you! Fixed